code/modules/jobs/job_types/prisoner.dm

```diff
@@ -0,0 +1,25 @@
+/datum/job/prisoner
+	title = "Prisoner"
+	flag = PRISONER
+	department_head = list("Head of security")
```

Comment:
huh?

Reply:
"You dont have any control over me, security man! You're not the HoS!"
- John Tiderton, shortly before getting his legs amputated by the warden

```suggestion
	department_head = list("the security team")
```